File: src/test/java/com/jesperancinha/service/containers/RunwayContainerTest.java
package com.jesperancinha.service.containers;

import com.jesperancinha.service.pojos.Runway;
import org.junit.Test;

import java.util.List;

import static org.hamcrest.Matchers.hasSize;
import static org.hamcrest.Matchers.notNullValue;
import static org.junit.Assert.assertThat;

/**
 * Created by joaofilipesabinoesperancinha on 08-08-16.
 */
public class RunwayContainerTest {
    @Test
    public void getRunways() throws Exception {
        final RunwayContainer runwayContainer = new RunwayContainer();

        List<Runway> runways = runwayContainer.getRunways();
        Runway runway = runways.get(0);

        assertThat(runways, hasSize(39537));
        assertThat(runway.getId(), notNullValue());
    }
}

File: src/test/kotlin/org/jesperancinha/airports/containers/RunwayContainerTest.kt
package org.jesperancinha.airports.containers

import org.hamcrest.Matchers
import org.jesperancinha.airports.pojos.Runway
import org.junit.Assert
import org.junit.Test

/**
 * Created by joaofilipesabinoesperancinha on 08-08-16.
 */
class RunwayContainerTest {
    @get:Throws(Exception::class)
    @get:Test
    val runways: Unit
        get() {
            val runwayContainer = RunwayContainer()
            val runways: List<Runway> = runwayContainer.runways
            val runway = runways[0]
            Assert.assertThat(runways, Matchers.hasSize(39537))
            Assert.assertThat(runway.id, Matchers.notNullValue())
        }
}